app/src/main/java/com/babylon/wallet/android/presentation/status/assets/nonfungible/NonFungibleAssetDialogContent.kt

```diff
@@ -380,7 +382,40 @@ private fun ClaimNFTInfo(
                     )
                 }
             }
+
             null -> {}
         }
     }
 }
+
+@Composable
+fun approximateClaimTimeText(approximateClaimMinutes: Long): String {
+    val hours = TimeUnit.MINUTES.toHours(approximateClaimMinutes)
+    val days = TimeUnit.MINUTES.toDays(approximateClaimMinutes)
+
+    return when { // TODO crowdin
+        days > 0 -> {
+            if (days == 1L) {
+                "$days day or less"
+            } else {
+                "$days days or less"
+            }
+        }
+
+        hours > 0 -> {
+            if (hours == 1L) {
+                "$hours hour or less"
+            } else {
+                "$hours hours or less"
+            }
+        }
+
+        else -> {
+            if (approximateClaimMinutes == 1L) {
+                "$approximateClaimMinutes minute or less"
+            } else {
+                "$approximateClaimMinutes minute or less"
```

Comment:
here the text should be "$approximateClaimMinutes minutes or less"

Reply:
fixed